### PR TITLE
Fixed validation errors phenopackets

### PR DESCRIPTION
**Pull request by frehburg**

### CODE REVIEW

**Comment by frehburg** [comment]
closes #148

**Comment by frehburg** [comment]
@ielis Please check if the phenopackets in `example-phenopackets-from-synthetic-data` now satisfy the criteria that you expect. They do not throw any errors on validation anymore

File: ERKER2Phenopackets/data/out/phenopackets/example-phenopackets-from-synthetic-data/0.json [hunk 1]
```diff
@@ -36,16 +56,12 @@
                 "expressions": [
                   {
                     "syntax": "hgvs",
-                    "value": "NP_005903.2:p.Thr112Met"
-                  },
-                  {
-                    "syntax": "hgvs",
-                    "value": "NM_005912.3:c.307G>A"
+                    "value": "NP_005903.2:p.Ile251Leu"
```
Comment (ielis) [inline ERKER2Phenopackets/data/out/phenopackets/example-phenopackets-from-synthetic-data/0.json:59]: There is a small issue in the protein HGVS string.

The [HGVS Protein recommendation](https://varnomen.hgvs.org/recommendations/protein/variant/substitution/) prescribes putting predicted consequences into parentheses:

> predicted consequences, i.e. without experimental evidence (no RNA or protein sequence analysed), should be given in parentheses, e.g. p.(Arg727Ser)

Correct me if I'm wrong, but the protein effects are predicted in your data (the protein variants were not determined experimentally, e.g. with proteomic methods). Therefore, you should use

```
NP_005903.2:p.(Ile251Leu)
```
for the example above as well as (probably) for the other phenopackets.

File: ERKER2Phenopackets/data/out/phenopackets/example-phenopackets-from-synthetic-data/0.json [hunk 2]
```diff
@@ -55,7 +71,7 @@
     }
   ],
   "metaData": {
-    "created": "2023-09-22T00:00:00Z",
+    "created": "2023-09-26T00:00:00Z",
     "createdBy": "P. Robinson, MD, D. Danis, PhD, A. Graefe, F. Rehburg",
```
Comment (ielis) [inline ERKER2Phenopackets/data/out/phenopackets/example-phenopackets-from-synthetic-data/0.json:75]: @frehburg @graefea it's flattering but I don't think there is any need to add me and Peter here at this point. :)

**Review by ielis — APPROVED** [review]
Hi,
the phenopackets indeed pass the validation with phenopacket-tools. I have a few minor comments, but in overall this looks good!

**Review by graefea — APPROVED** [review]

**Pull request merged**